plugin/trino-iceberg/pom.xml

```diff
@@ -238,6 +257,40 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>5.1</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.httpcomponents.core5</groupId>
+            <artifactId>httpcore5</artifactId>
+            <version>5.1.1</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
Why do we need this? We have our own S3 implementation, so we shouldn't be using the Iceberg AWS code.

This should go in the root POM with the other Iceberg dependencies.

Reply:
I've removed iceberg-aws and sdk-v2 dependencies, but this may limit the use of some features, but we can revisit later.